index.js

```diff
@@ -92,32 +143,28 @@ async function run(env, eventPayload, fs, core) {
       }
 
       return [...prev, curr];
-    }, [])
-    .map(([key, ...lines]) => {
-      const checkListValue = lines.find((line) => Array.isArray(line));
-      const value = checkListValue
-        ? toValue(checkListValue)
-        : toValue(...lines);
-
-      return [toKey(key), value];
-    });
+    }, []);
 
-  result.forEach(([key, value]) => {
-    core.setOutput(`issueparser_${key}`, value);
+  result = result.map(([key, ...lines]) => {
+    const checkListValue = lines.find((line) => Array.isArray(line));
+    const value = checkListValue ? toValue(checkListValue) : toValue(...lines);
+
+    return [toKey(key), value];
   });
 
+  result = toObject(result);
+  core.setOutput(`issueparser`, result);
```

Comment:
I wonder if this change was intended. Right now every form value becomes its own output e.g `issueparser_ email = hello@me.com`. However, this change will result in a single output named `issueparser` which will include the entire parsed response as a json string, which is the same as the current existing `jsonString` output.  

Original I added this output per form value, to avoid the write/read file dance in case only value is needed. I think that way it is easier to work with that a json-string that needs to be parse ... What do you think @elhmn?

Reply:
Also interesting that none of the unit tests was catching that issue. The mock is looking for `issueparser_` ... I'll look into this